### ISSUE TITLE
Consider adding a license to the repo

### ISSUE THREAD

**Issue by john-h-k** [d3dcoder/d3d12book#20]
Currently this is unlicensed, meaning
> no one may reproduce, distribute, or create derivative works from your work
It seems providing a license or an explicit unlicense would be beneficial to anyone wanting to use the code here

**Comment by MasterDrake** [comment]
Yeah, seems weird to write a renderer book, publish the code and not add a license
What we can do with the code?

**Comment by AntMaster7** [comment]
I'm also looking into finding license details.
